mm2src/rpc/get_public_key_hash.rs

```diff
@@ -0,0 +1,36 @@
+use common::{mm_ctx::MmArc, mm_error::MmError, HttpStatusCode};
+use derive_more::Display;
+use http::StatusCode;
+use serde_json::Value as Json;
+
+#[derive(Serialize)]
+pub struct GetPublicKeyHashResponse {
+    public_key_hash: String,
```

Comment:
It's preferred to use the right serializable type for this. Please use the right type from here https://github.com/KomodoPlatform/atomicDEX-API/blob/dbacb816c3ece306628ca7517ef57c29036e0b1e/mm2src/mm2_bitcoin/rpc/src/v1/types/hash.rs#L162-L164